saleor/graphql/core/tests/test_graphql.py

```diff
@@ -52,7 +51,7 @@ def test_jwt_middleware(client, admin_user):
     response = api_client_post(data={"query": user_details_query})
     repl_data = response.json()
     assert response.status_code == 200
-    assert isinstance(response.wsgi_request.user, AnonymousUser)
+    assert response.wsgi_request.user == None  # noqa: E711
```

Comment:
why not `is None`?

Reply:
Because `response.wsgi_request.user` is lazy. So if I compare it to `is None` it will be False.